tests/clock/jobCleanup.js

```diff
@@ -99,7 +110,12 @@ describe('tests/clock/jobCleanup.js >', () => {
       expectNJobs(jobCount)
       .then(() => jobCleanup.execute(batchSize, delay))
       .then(() => expectNJobs(expectedCount))
-      .then(done).catch(done);
+      .then(() => {
+        expect(activityLogUtil.printActivityLogString.calledOnce)
```

Comment:
Nice! Clever way to check for the logging. I was just using winston before but this seems cleaner

Reply:
Downside is that it doesn't check content of the logs. Can you add a test to make sure the correct fields are being logged?

Reply:
Sure! Good point.

Will added that. 👊🏻